biggus/__init__.py

```diff
@@ -824,6 +832,47 @@ def masked_array(self):
         return array
 
 
+def _use_indexed_array(keys):
+    return ((isinstance(keys, tuple) and any(key is None for key in keys)) or
+            keys is None)
```

Comment:
Just a stylistic thing ... but since `np.newaxis is None` would it add any clarity to use `np.newaxis` as a convenience, rather than `None`, to provide a massive hint to the developer/reader as to the purpose of `IndexedArray` ? Just a thought ...